.github/workflows/build-images-base.yaml

```diff
@@ -69,7 +69,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Check out code
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
+      - name: Read release string version
+        id: release
+        run: |
+          version=`make read-release-version`
+          echo version=$version >> $GITHUB_OUTPUT
```

Comment:
I wouldn't put this here... rather pass it as a IMG_TAG from the caller workflow. Also, I've seen that you are adding the leading `v` in the make target, that would mean a _tag_ not a _version_

Reply:
But `IMG_TAG` may have many tags, and we only want one, which is the one from the Makefile. That will be the one being shown in the logs.

>I've seen that you are adding the leading v in the make target, that would mean a tag not a version

A tag is something for the docker images. This is a release version. To avoid confusion, are you happy with `release_version` instead of `version`?

Reply:
`IMG_TAGS` _might_ have many, but you can just pass the one you want: https://github.com/Kuadrant/limitador-operator/pull/169/files#diff-684aa95ad40ba272fd193cfaf4a40d0b79ba86fe71ba222ed0bd09ce574a226fR58

Reply:
So, you do not want to read the release version from the makefile. Fair enough for this workflow not meant for release tags.

Instead of `IMG_TAGS` that are tags of the docker images, are you happy to set from `inputs.operatorVersion`?

Reply:
What I mean is that you can pass the version/tag that can be read from the Makefile to the `inputs.operatorVersion` and `inputs.operatorTag` like in here https://github.com/Kuadrant/limitador-operator/blob/main/.github/workflows/build-images-scheduled.yaml#L13-L14 . Maybe I'm missing something but the base workflow was meant to not change, we might simplify it but not add new things, that's why the other workflows (for branches, scheduled, etc..) exists